build/root/Makefile.generated_files

```diff
@@ -131,7 +131,7 @@ ifeq ($(DBG_MAKEFILE),1)
     $(warning ***** finding all +k8s:prerelease-lifecycle-gen tags)
 endif
 PRERELEASE_LIFECYCLE_DIRS := $(shell                                             \
-    grep --color=never -l '+k8s:prerelease-lifecycle-gen=' $(ALL_K8S_TAG_FILES)  \
+    grep --color=never -l '+k8s:prerelease-lifecycle-gen' $(ALL_K8S_TAG_FILES)  \
```

Comment:
what happens if there are prerelease-lifecycle-gen tags but not prerelease-lifecycle-gen=true? is prerelease-lifecycle-gen=true even required now?

Reply:
> what happens if there are prerelease-lifecycle-gen tags but not prerelease-lifecycle-gen=true? is prerelease-lifecycle-gen=true even required now?

Yeah.  The generator looks for it.  This is a superset selection of all possibles. However, it still didn't do what I want.